package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vis-network",
-  "version": "5.1.1",
+  "version": "0.0.0-no-version",
```

Comment:
why this?

Reply:
We will release through semantic-release which doesn't update this. In other words we would have 5.1.1 forever.